sdks/python/apache_beam/examples/inference/sklearn_japanese_housing_regression.py

```diff
@@ -0,0 +1,161 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""A pipeline that uses RunInference API on a regression about housing prices.
+
+This example uses the japanese housing data from kaggle.
+
+Since the data has missing fields, this example illustrates how to split
+data and assign it to appropriate models. The predictions are then recombined.
```

Comment:
different models are trained on different features of data right? may be we can mention something like that in the comments as well?
```suggestion
data and assign it to the models that are trained on different subsets of features. The predictions are then recombined.
```